src/main/java/com/nguyenquyhy/discordbridge/discordcommands/CommandHandler.java

```diff
@@ -0,0 +1,106 @@
+package com.nguyenquyhy.discordbridge.discordcommands;
+
+import java.util.Optional;
+import org.spongepowered.api.Sponge;
+import org.spongepowered.api.command.CommandSource;
+import org.spongepowered.api.entity.living.player.Player;
+import org.spongepowered.api.text.Text;
+import org.spongepowered.api.text.format.TextColors;
+
+import com.nguyenquyhy.discordbridge.DiscordBridge;
+import com.nguyenquyhy.discordbridge.models.ChannelConfig;
+import com.nguyenquyhy.discordbridge.models.GlobalConfig;
+import com.nguyenquyhy.discordbridge.utils.ErrorMessages;
+import com.nguyenquyhy.discordbridge.utils.DiscordUtil;
+import de.btobastian.javacord.DiscordAPI;
+import de.btobastian.javacord.entities.Channel;
+import de.btobastian.javacord.entities.User;
+import de.btobastian.javacord.entities.message.Message;
+import de.btobastian.javacord.entities.permissions.PermissionState;
+import de.btobastian.javacord.entities.permissions.PermissionType;
+import de.btobastian.javacord.entities.permissions.Permissions;
+import de.btobastian.javacord.entities.permissions.Role;
+
+public class CommandHandler {
+
+    private static String id = "discordbridge";
+
+    public static void handle(Optional<Player> player, String substring) {

```

Comment:
This method isn't used yet, its for future support of In Game commands, such as allowing Admin's online to use !ban on a discord user.